syft/workers/worker_support.py

```diff
@@ -0,0 +1,18 @@
+from syft import dependency_check
+
+from syft.workers.base import BaseWorker
+
+from syft.frameworks.crypten.worker_support import add_support_to_worker, remove_support_from_worker
+
+supported_frameworks = {}
+
+if dependency_check.crypten_available:
+    supported_frameworks["crypten"] = [add_support_to_worker, remove_support_from_worker]
+
+
+def add_support(worker: BaseWorker, framework: str) -> None:
+    supported_frameworks[framework][0](worker)
```

Comment:
I'm not sure this scales: maybe you could replace the list [add_support_to_worker, remove_support_from_worker] by a dict with "standard" keys, so that if you need to has more on them you don't mess up with indices

Reply:
Done